client/components/PhotogridComponents/Photogrid.jsx

```diff
@@ -20,6 +20,11 @@ const Big = styled.div`
   grid-area: 1 / 1 / 3 / 3;
   border-top-left-radius: 15px;
   border-bottom-left-radius: 15px;
+  &:hover {
+    -webkit-filter: grayscale(50%);
```

Comment:
oh I am interested in what this does

Reply:
It greys out something when hovering it!